src/installer/tests/Directory.Build.targets

```
@@ -127,5 +127,14 @@
     </ItemGroup>
   </Target>
 
+  <!-- Properties used by runsettings.targets -->
+  <PropertyGroup>
+    <RunSettingsDotNetHostPath>$(DotNetTool)</RunSettingsDotNetHostPath>
+    <RunSettingsTestCaseFilter>$(TestCaseFilter)</RunSettingsTestCaseFilter>
+  </PropertyGroup>
+
+  <Import Project="$(RepositoryEngineeringDir)testing\runsettings.targets"
+          Condition="'$(IsTestProject)' == 'true' and '$(ContinuousIntegrationBuild)' != 'true'" />
```

Is there a strong reason why you don't want to run this logic in CI? Generating the runsettings file shouldn't be noticeable and while the file isn't used in CI, it would add protection to make sure this code doesn't regress.

---

Nope, no strong reason - will remove the condition.
```suggestion
          Condition="'$(IsTestProject)' == 'true'" />
```